test/python/common.py

```diff
@@ -308,42 +356,20 @@ def requires_qe_access(func):
     """
 
     @functools.wraps(func)
-    def _wrapper(*args, **kwargs):
-        if SKIP_ONLINE_TESTS:
+    def _wrapper(self, *args, **kwargs):
+        if TEST_OPTIONS['skip_online']:
             raise unittest.SkipTest('Skipping online tests')
 
         # Cleanup the credentials, as this file is shared by the tests.
         from qiskit.wrapper import _wrapper
```

Comment:
Import the `_wrapper` with a different name to avoid name collision:
```python
from qiskit.wrapper import _wrapper as qiskit_wrapper
```

Reply:
a49e769